pkg/virt-launcher/virtwrap/network/network.go

```diff
@@ -144,5 +144,5 @@ func isSecondaryMultusNetwork(net v1.Network) bool {
 }
 
 func newpodNIC(cacheFactory cache.InterfaceCacheFactory) podNIC {
-	return &podNICImpl{cacheFactory: cacheFactory, handler: Handler}
+	return &podNICImpl{cacheFactory: cacheFactory, handler: &NetworkUtilsHandler{}}
```

Comment:
I find it weird that a vmi/pod with multiple nics will have multiple instances of `NetworkUtilsHandler`.

Reply:
I have prome it as podnic factory method arg, now the instance is reused between pod nics.